src/hardwareio/IMUCalibDumper/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#################################################################################
+#                                                                               #
+# Copyright (C) 2019 Fondazione Istituto Italiano di Tecnologia (IIT)           #
+# All Rights Reserved.                                                          #
+#                                                                               #
+#################################################################################
+
+# @authors: Leandro de Souza Rosa <leandro.desouzarosa@iit.it>
+
+project(IMUDataDumper)
+
+file(GLOB source src/*.cpp)
+file(GLOB header include/*.h)
+
+include_directories(
+    ${PROJECT_SOURCE_DIR}/include
+)
+
+add_executable(${PROJECT_NAME}
+    ${source}
+    ${header}
```

Comment:
I don't see the need for the project to be split into 3 files when a single `.ccp` file could contain everything. We aren't going to link to the `.h` file in any other project - can we merge it all into the 1 file?